marathon/src/main/scala/io/buoyant/marathon/v2/AppIdNamer.scala

```diff
@@ -0,0 +1,119 @@
+package io.buoyant.marathon.v2
```

Comment:
yeah, i originally wrote it that way. @olix0r mentioned that it's preferable to tie AppIdNamer to a specific API version.